src/GraphQLParser/AST/Enums.cs

```diff
@@ -25,6 +25,7 @@ public enum ASTNodeKind
         NamedType,
         ListType,
         NonNullType,
+        Null,
```

Comment:
I think the only suggestion I have is for this to be `NullValue` vs. just `Null`.